services/media/service/src/ingest/handlers/video-succeeded-handler.db.spec.ts

```diff
@@ -95,7 +84,6 @@ describe('VideoSucceededHandler', () => {
 
   afterEach(async () => {
     await ctx.truncate('ingest_documents');
-    payloads = [];
   });
 
   afterAll(async () => {
```

Comment:
onMessage and onMessageFailure descriptions could be renamed to handleMessage and handleErrorMessage.